src/reuse/_main.py

```diff
@@ -205,6 +206,15 @@ def parser() -> argparse.ArgumentParser:
         help=_("print the project's bill of materials in SPDX format"),
     )
 
+    add_command(
+        subparsers,
+        "supported-licenses",
+        supported_licenses.add_arguments,
+        supported_licenses.run,
+        help=_("list all supported SPDX licenses"),
+        aliases=["supported-licences"],
```

Comment:
I've added this one because it drives me crazy otherwise :upside_down_face:  Shouldn't hurt anyone.